contracts/Logger.sol

```diff
@@ -74,16 +74,6 @@ contract Logger {
         emit DepositQueued(accountID, data);
     }
 
-    event DepositLeafMerged(bytes32 left, bytes32 right, bytes32 newRoot);
-
-    function logDepositLeafMerged(
```

Comment:
I removed the event DepositLeafMerged to save gas because the client only cares about the ready subtree event.